src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -64,28 +63,30 @@ public void execute_validIndexFilteredList_success() {
     }
 
     @Test
-    public void execute_invalidIndexFilteredList_throwsCommandException() {
+    public void execute_invalidNameFilteredList_throwsCommandException() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
-        Index outOfBoundIndex = INDEX_SECOND_PERSON;
+        Person person2ToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
         // ensures that outOfBoundIndex is still in bounds of address book list
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getPersonList().size());
+        assertTrue(model.getAddressBook().getPersonList().contains(person2ToDelete));
 
-        DeleteCommand deleteCommand = new DeleteCommand(outOfBoundIndex);
+        DeleteCommand deleteCommand = new DeleteCommand("WHATever it is");
```

Comment:
please put this in a final variable like `final String nonExistantName = "WHATever it is";`